Todo.WillowBear/Todo.WillowBear/Data/TodoDbContext.cs

```diff
@@ -0,0 +1,160 @@
+using Microsoft.EntityFrameworkCore;
+using Todo.WillowBear.Models;
+using Serilog;
+
+namespace Todo.WillowBear.Data
+{
+    public class TodoDbContext : DbContext
+    {
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlite("Data Source=TodoList.db");
```

Comment:
As I understand (and I didn't make it clear in the requirements), you created the tables using as script directly against the DB, correct? If that's the case could you please send me the script you used to create the DB?

Thanks!

Reply:
I didn't! The following code in TodoItemsConfiguration.cs created the table

public class TodoItemConfiguration : IEntityTypeConfiguration<TodoItem>
    {
        public void Configure(EntityTypeBuilder<TodoItem> builder)
        {
            builder.ToTable("TodoItems");
            builder.HasKey(x => x.Id);
            builder.Property(x => x.Title).IsRequired().HasMaxLength(20);
            builder.Property(x => x.IsDone).IsRequired().HasDefaultValue(0);
            builder.Property(x => x.CreatedAt).IsRequired().HasDefaultValueSql("datetime('now')");
            builder.Property(x => x.DueDate);
        }
    }

Reply:
Should I remove that file and by directly against the Db do you mean an SQL script within SQL management studio?